Modules/_asynciomodule.c

```diff
@@ -1969,6 +1974,17 @@ _asyncio_Task___init___impl(TaskObj *self, PyObject *coro, PyObject *loop)
     Py_INCREF(coro);
     Py_XSETREF(self->task_coro, coro);
 
+    if (name == Py_None) {
+        name = PyUnicode_FromFormat("Task-%llu", ++task_name_counter);
+    } else {
+        name = PyObject_Str(name);
+    }
+
```

Comment:
Please remove this new line to make it easier for the reader to see that we handle the case of `name==NULL`